framework/include/transfers/Transfer.h

```diff
@@ -77,8 +77,8 @@ class Transfer : public MooseObject, public SetupInterface, public Restartable
   ///@{
   /// The current direction that this Transfer is going in.
   /// direction() is to be deprecated for currentDirection()
-  int direction() { return _direction; }
-  int currentDirection() { return _current_direction; }
+  MooseEnum direction() { return _direction; }
+  MooseEnum currentDirection() { return _current_direction; }
```

Comment:
I remember trying this and there's a failure downstream. I might have removed it when I reworked to use direction less though

Reply:
oh, hopefully this will be ok. MooseEnum can be implicitly convert to int. I do not need this. I was printing direction so made this change, but later realized I do not need that print. I kept this in this PR, but if it gives us trouble, I can drop it.